src/Auth/Process/Authorize.php

```diff
@@ -147,7 +161,20 @@ public function __construct(array $config, $reserved)
                     ));
                 }
             }
-            $this->valid_attribute_values[$attribute] = $values;
+
+            // For backward compatibility, if no spEntityIDs were found,
+            // store the values directly in the old format
+            if ($spEntityIDs === null) {
```

Comment:
This if/else block is unnecessary...
`$spEntityIDs` is set to `null` as a default on line 142 and is only set to something else if it's an array.

Reply:
Thanks. True, fixed.